### PR TITLE
Add simplified scripts for reflashing test machine to UC

### PR DESCRIPTION
**Pull request by valentindavid**

### CODE REVIEW

**Comment by valentindavid** [comment]
See https://github.com/snapcore/upower-snap/pull/4

File: lib/build-image.sh [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        ;;
+esac
+
+case "${ssh_user}" in
+    root)
+        # Some spread test instances (like GCE) expect root to be
+        # used. We cannot set a password for root with extrausers. So
+        # we need to bind mount a modified shadow (which is read-only)
+        # to set the root password
+        days="$(($(date +%s)/60/60/24))"
```
Comment (alfonsosanchezbeato) [inline lib/build-image.sh]: It looks like `days` is not being used

File: lib/build-image.sh
Comment (alfonsosanchezbeato) [inline lib/build-image.sh:1]: Please add a copyright header (I know we don't have it yet in other files in this repo but it is probably a good time to start fixing that :slightly_smiling_face:)

File: lib/flash.sh
Comment (alfonsosanchezbeato) [inline lib/flash.sh:1]: Same: re copyright

File: lib/build-image.sh [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+# any version.
+
+set -eu
+
+TMPDIR="$(mktemp -d)"
+cleanup() {
+    rm -rf "${TMPDIR}"
+}
+
+trap cleanup EXIT
```
Comment (alfonsosanchezbeato) [inline lib/build-image.sh:35]: EXIT sigspec in `dash` does not match the one in `bash`, so cleanup() is not run on SIGTERM/SIGINT, I suggest to change to using bash in the script.

File: lib/build-image.sh [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+cat <<EOF >>"${TMPDIR}/pc/gadget.yaml"
+defaults:
+  system:
+    service:
+      console-conf:
+        disable: true
+EOF
+
+
```
Comment (alfonsosanchezbeato) [inline lib/build-image.sh]: (big) nitpick: double empty line

File: lib/build-image.sh
Comment (valentindavid) [reply lib/build-image.sh:1]: Should we use GPL-3? It seems it is what snapcore/snapd-testing-tools uses. It should probably be the same.

Comment (valentindavid) [reply lib/build-image.sh:1]: Put GPL-3 header. We will need to add COPYING in the repo. But we should maybe do that in a PR that fixes all the other files.

File: lib/build-image.sh (hunk 2 above)
Comment (valentindavid) [reply lib/build-image.sh:35]: Changed to `bash`.

File: lib/flash.sh
Comment (alfonsosanchezbeato) [inline lib/flash.sh:1]: Please change the script to using bash, for homogeneity (all repo scripts use it)

**Review by alfonsosanchezbeato — COMMENTED** [review]
Thanks for the changes, just one final thing

File: lib/flash.sh
Comment (valentindavid) [reply lib/flash.sh:1]: Done.

**Review by alfonsosanchezbeato — APPROVED** [review]
Thanks for the changes!

**Pull request merged**